editor/src/components/editor/store/dispatch.tsx

```diff
@@ -250,8 +250,9 @@ function maybeRequestModelUpdate(
         codeNeedsPrinting(file.fileContents.revisionsState) &&
         isParseSuccess(file.fileContents.parsed)
       ) {
+        const stripUIDs = process.env.NODE_ENV === 'production'
```

Comment:
I'm going to change this to use the `PRODUCTION_ENV` const we declare in `env-vars.ts` actually